qiskit/test/mock/utils/fake_backend_builder.py

```diff
@@ -0,0 +1,348 @@
+            'name': 'measure',
+            'qubits': list(range(self.n_qubits)),
+            'sequence': measure_command_sequence
+        }).to_dict()
+
+        cmd_def = [measure_command]
+
+        for i in range(self.n_qubits):
+            cmd_def += [
+                Command.from_dict({
+                    'name': 'u1',
+                    'qubits': [i],
+                    'sequence': [PulseQobjInstruction(name='fc', ch='d{}'.format(i),
+                                                      t0=0, phase='-P0').to_dict()]
+                }).to_dict(),
+                Command.from_dict({
+                    'name': 'u2',
+                    'qubits': [i],
+                    'sequence': [PulseQobjInstruction(name='fc', ch='d{}'.format(i),
+                                                      t0=0, phase='-P1').to_dict(),
+                                 PulseQobjInstruction(name='test_pulse_4',
+                                                      ch='d{}'.format(i), t0=0).to_dict(),
+                                 PulseQobjInstruction(name='fc', ch='d{}'.format(i),
+                                                      t0=0, phase='-P0').to_dict()]
+                }).to_dict(),
+                Command.from_dict({
+                    'name': 'u3',
+                    'qubits': [i],
+                    'sequence': [PulseQobjInstruction(name='test_pulse_3',
+                                                      ch='d{}'.format(i), t0=0).to_dict()]
+                }).to_dict()
+            ]
+
+        for couple in self.cmap:
+            qubit1, qubit2 = couple
+            cmd_def += [
+                Command.from_dict({
+                    'name': 'cx',
+                    'qubits': [qubit1, qubit2],
+                    'sequence': [PulseQobjInstruction(name='test_pulse_1',
+                                                      ch='d{}'.format(qubit1),
+                                                      t0=0).to_dict(),
+                                 PulseQobjInstruction(name='test_pulse_2',
+                                                      ch='u{}'.format(qubit1),
+                                                      t0=10).to_dict(),
+                                 PulseQobjInstruction(name='test_pulse_1',
+                                                      ch='d{}'.format(qubit2),
+                                                      t0=20).to_dict(),
+                                 PulseQobjInstruction(name='fc', ch='d{}'.format(qubit2),
+                                                      t0=20, phase=2.1).to_dict()]
+                }).to_dict()
+            ]
+
+        return PulseDefaults.from_dict({
+            'qubit_freq_est': meas_freq_est,
+            'meas_freq_est': qubit_freq_est,
+            'buffer': buffer,
+            'pulse_library': pulse_library,
+            'cmd_def': cmd_def
+        })
+
+    def dump(self, folder: str):
```

Comment:
Perhaps this method should be named `save` and if it exists maybe there should be a classmethod `load` that loads from a dumped folder?